packages/components/addon/components/hds/form/checkbox/base.hbs

```diff
@@ -0,0 +1 @@
+<input type="checkbox" class={{this.classNames}} ...attributes value={{@value}} />
```

Comment:
[question] Do you think that we should ensure that something exists for `@value`?

Reply:
Probably not. Why? We have the same approach for all the other controls. Something we're missing/not considering?